bin/nyoom

```diff
@@ -23,7 +23,7 @@ case "${1:-}" in
     nvim --version | sed '/^$/,$d'
     printf "\nClearing cache\n";
     rm -rf ~/.cache/nvim/
```

Comment:
Does it make sense to apply the same to the `.cache` directory?

Or it is semantically wrong since `cache` does not relate to `config`?

Reply:
It does, we could use `XDG_CACHE_HOME` for that, I'll add that change.

Reply:
https://wiki.archlinux.org/title/XDG_Base_Directory

Reply:
Looks good!

Reply:
I've found one more :)
In the 16th and 17th lines `~/.local/share` is used, which is `$XDG_DATA_HOME`

Given we've got 3 of these variables, and one of them is to be used twice,
I would define all of them at the top of the file. It will make the places where they are used now, more concise and a bit easier to read
``` bash
$XDG_CONFIG_HOME=...
$XDG_CACHE_HOME=...
$XDG_DATA_HOME=...
```

Of course this is just a suggestion how to make things even fancier. But it is already definitely better than it was before.

Reply:
That would work just fine as well, I assume you mean something akin to:
```bash
setup_envs() {
  CONFIG_PATH="${XDG_CONFIG_HOME:-${HOME}/.config}/nvim"
  CACHE_PATH="${XDG_CACHE_HOME:-${HOME}/.cache}/nvim"
  DATA_HOME="${XDG_DATA_HOME:-${HOME}/.local/share}/nvim"
}

setup_envs "${@}"
```

Reply:
Yeah, to me it looks good as well.

But I do not get it, why do you pass `"${@}"` to `setup_envs` :thinking: 

And of course the naming should be consistent (`DATA_HOME` -> `DATA_PATH`).

Reply:
Or maybe
```
NVIM_CONFIG
NVIM_CACHE
NVIM_DATA
```

I'm not insisting on anything of it. Just trying to suggest some option, which you may like more

Reply:
I've switched to the env's and added a conditional on the install so the `install` doesn't stop if `packer` is already installed.